R/create_covariates.R

```diff
@@ -0,0 +1,1264 @@
+#' Covariate SAGA raster covarates from base digital terrain dataset.
+#'
+#'#' Takes a dem and via SAGA GIS generates raster covariates at the resolution of
+#' the dem.
+#'
+#' Depending on your system the path to `saga_cmd` may need to be specified.
+#'
+#' @param dtm a character or file path location to raster with elevation data.
+#'        Outputs will be produced at the same resolution and extent as input dtm
+#' @param saga_path a character of the file to the SAGA directory on the analysts system.
+#'          On linux systems with SAGA GIS installed Use `SAGApath = ""`
+#' @param out_dir file.path. Directory where covariates will be written. If not
+#'     specified uses the default from the `fid` folder structure. If files already
+#'     exists they will NOT be overwritten.
+#' @param layers character vector. Covariates to be created. Default is \code(`all`)
+#'  - call \code(`PEMprepr::layer_options`) for full list.
```

Comment:
```suggestion
#' @param layers character vector. Covariates to be created. Default is `"all"`
#'  - call `PEMprepr::layer_options` for full list.
```

Reply:
Will keep this for now, but might depend on how we rework the create_covariates().